pkg/compliance/k8sconfig/loader.go

```diff
@@ -0,0 +1,778 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package k8sconfig
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/ed25519"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/shirou/gopsutil/v3/process"
+	"gopkg.in/yaml.v3"
+)
+
+const version = "202305"
+
+const (
+	k8sManifestsDir   = "/etc/kubernetes/manifests"
+	k8sKubeconfigsDir = "/etc/kubernetes"
+)
+
+type K8sNodeConfig struct {
```

Comment:
Maybe move the types to a `type.go` file.

Reply:
done